code/src/App.js

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { HappyThought } from 'components/HappyThought'
+import { HappyForm } from 'components/HappyForm';
```

Comment:
I actually learnt from my last code review that the outer dependencies (those located in node_modules) should be separated from local dependencies (all local files that we created like: components, functions, data such as arrays) with one empty space above/under each other. See the example under! 😁

import React, { useEffect, useState } from 'react'

import { HappyThought } from 'components/HappyThought'
import { HappyForm } from 'components/HappyForm';